packages/admin-ui-components/src/statementsTable/statementsTableContent.tsx

```diff
@@ -276,19 +276,35 @@ export const StatementTableCell = {
   ),
 };
 
-export const StatementLink = (props: {
+interface StatementLinkProps {
   statement: string;
   app: string;
   implicitTxn: boolean;
   search: string;
-}) => {
+  anonStatement?: string;
+}
+
+// StatementLinkTarget returns the link to the relevant statement page, given
+// the input statement details.
+export const StatementLinkTarget = (props: StatementLinkProps) => {
```

Comment:
Would this introduce problems in CC if we need to route to a different URL set?

Reply:
Looks like it should not. Current change is just moving out basically the same url forming logic to separate function. But I will double check it when will be working on CC integration/